controller/backup_controller.go

```diff
@@ -235,7 +235,7 @@ func (bc *BackupController) reconcile(backupName string) (err error) {
 			return err
 		}
 
-		if backupTarget.DeletionTimestamp == nil && backupTarget.Spec.BackupTargetURL != "" &&
```

Comment:
Please retain the check `backupTarget.DeletionTimestamp == nil`. This prevents the remote files from being cleaned up if users delete the CR by accident.

Reply:
But, we'll auto salvage the default BackupTarget CR, isn't?

Reply:
OK. When this line is executed, the CR won't be `nil`.

Reply:
Also, we have check the BackupTarget CR is not found and return nil before this line.
And since we remove the finalized of BackupTarget CR, so the deletion time stamp always nil from my understanding.